modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
I vote for getting rid of `time`. Let's not encourage users to rush puzzles, the opposite is desired.